charts/px-backup/templates/pxcentral-onprem.yaml

```diff
@@ -0,0 +1,87 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: pxcentral-admin-user
+  namespace: {{ .Values.namespace }}
```

Comment:
throughout the chart, I see it's referencing the namespace from values. Do we need this?

The templates shouldn't be needing to have a namespace field since helm will automatically create all manifests in the namespace specified in the helm install command. 

Reference: https://github.com/helm/helm/issues/5465

Reply:
It is required. Without it `helm template --namespace` won't render the templates for the correct namespace

Reply:
what if the namespace is not provided to helm install command?